src/main/java/teammates/sqllogic/core/CoursesLogic.java

```diff
@@ -120,8 +123,8 @@ public void deleteCourseCascade(String courseId) {
 
         // TODO: Migrate after other Logic classes have been migrated.
```

Comment:
Is this comment still relevant? i.e. what other logic classes we need to complete for this

Reply:
i think we can migrate this already, let's do this in a separate PR